apps/google-analytics-4/lambda/serverless.yml

```diff
@@ -0,0 +1,22 @@
+service: google-analytics
+frameworkVersion: '3'
+
+plugins:
+  - serverless-offline
+
+custom:
+  serverless-offline:
+    httpPort: 8080
+
+provider:
+  name: aws
+  runtime: nodejs16.x
```

Comment:
New service, let's use node 18 straight away? (if possible)

Reply:
Ah great, I see they just added this in November.

I will flag this for a follow up PR 👍